src/includes/class-alma-wc-refund-helper.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * Alma Alma refund helper
+ *
+ * @package Alma_WooCommerce_Gateway
+ */
+
+use Alma\API\RequestError;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Not allowed' ); // Exit if accessed directly.
+}
+
+/**
+ * Alma_WC_Refund_Helper
+ */
+class Alma_WC_Refund_Helper {
+
+	/**
+	 * Logger
+	 *
+	 * @var Alma_WC_Logger
+	 */
+	private $logger;
+
+	/**
+	 * __construct.
+	 */
+	public function __construct() {
+		$this->logger = new Alma_WC_Logger();
+	}
+
+	/**
+	 * Gets the amount to refund.
+	 *
+	 * @param integer $refund_id Refund id.
+	 * @param bool    $display Tells if amount is supposed to be used for calculation or display.
+	 * @return int|string
```

Comment:
avoid to give more than one responsibility to your method (S = Single Responsibility principles from SOLID)

Here you have to split your method in two and remove the `$display` parameter.

Then, when you have to call method with `$display === false`, just call `get_amount_to_refund()` that will return an `int`

And, when you have to call method with `$display === true`, call `get_price_to_refund()` that will return a `string` (for example)

Reply:
Right !
Done !